src/frontend/js/hooks/useMatchMedia.ts

```diff
@@ -34,4 +34,10 @@ const useMatchMedia = (query: string): boolean => {
   return matches;
 };
 
+const emCalc = (px: number) => {
+  return px / 16;
+};
+
+export const useMatchMediaLg = () => useMatchMedia(`(max-width: ${emCalc(992)}em)`);
```

Comment:
I feel like media width should be compute with px and not em.
Em are good for elements size and space, here we talk about the actual number of px of the screen.

Reply:
This is fixed I talked about it on Slack + @jbpenrath comment here: https://github.com/openfun/richie/pull/2355#discussion_r1576402747